galaxy.yml

```diff
@@ -8,16 +8,15 @@ namespace: crowdstrike
 name: falcon
 
 # The version of the collection. Must be compatible with semantic versioning
-version: 2.0.0
+version: 2.1.1
 
 # The path to the Markdown (.md) readme file. This path is relative to the root of the collection
 readme: README.md
 
 # A list of the collection's content authors. Can be just the name or in the format 'Full Name <email> (url)
 # @nicks:irc/im.site#channel'
 authors:
-  - Shawn Wells <shawndwells@gmail.com>
-  - Gabriel Alford <redhatrises@gmail.com>
+  - CrowdStrike <integrations@crowdstrike.com>
```

Comment:
```suggestion
  - CrowdStrike Solutions Architecture <integrations@crowdstrike.com>
```

Reply:
Since this is supported within partner&alliances vs broader CrowdStrike, we should probably put Solution Architecture here

Reply:
Okay... done